### ISSUE TITLE
Minikube & Hyper-V: Error copying temp iso file to destination

### ISSUE THREAD

**Issue by PierrickI3** [kubernetes/minikube#760]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Bug

**Minikube version** (use `minikube version`): 0.12.0

**Environment**:
- **OS**: Windows 2012 R2 x64
- **VM Driver**: Hyper-V
- **Install tools**: Copied minikube-amd64 to c:\windows\system32 and renamed it minikube.exe

**What happened**: An error occurs when running `minikube start --vm-driver=hyperv` from an administrative command line:

```
C:\Users\Administrator> minikube start --vm-driver=hyperv
Starting local Kubernetes cluster...
E1027 02:14:44.459093    3716 start.go:90] Error starting host: Error attempting to cache minikube iso from url: Error copying temp iso file to destination: rename C:\Users\Administrator\.minikube\cache\iso\.tmp-minikube-0.7.iso336906043 C:\Users\Administrator\.minikube\cache\iso\minikube-0.7.iso: The process cannot access the file because it is being used by another process.. Retrying.
E1027 02:14:44.465119    3716 start.go:96] Error starting host: Error attempting to cache minikube iso from url: Error copying temp iso file to destination: rename C:\Users\Administrator\.minikube\cache\iso\.tmp-minikube-0.7.iso336906043 C:\Users\Administrator\.minikube\cache\iso\minikube-0.7.iso: The process cannot access the file because it is being used by another process.
================================================================================
```

At the time, Hyper-V manager & chrome are the only running applications.

**What you expected to happen**:

Minikube iso image should have been cached

**How to reproduce it** (as minimally and precisely as possible):

minikube start --vm-driver=hyperv

**Comment by PierrickI3** [comment]
Also, here are the contents of the /iso folder:
![clipboard01](https://cloud.githubusercontent.com/assets/7140338/19762731/efb86038-9bef-11e6-9487-f75e070ef49d.jpg)

**Comment by jimmidyson** [comment]
Sorry about that - fixed in #756 which will be in next release.

**Comment by PierrickI3** [comment]
No problem. Thanks @jimmidyson
